bot/exts/evergreen/emoji_count.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import logging
+import random
+from typing import Dict, Optional
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class EmojiCount(commands.Cog):
+    """Command that give random emoji based on category."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    def embed_builder(self, emoji: dict) -> discord.Embed:
+        """Genrates embed with emoji name and count."""
+        embed = discord.Embed()
+        embed.color = Colours.orange
+        embed.title = "Emoji Count"
+        embed.timestamp = datetime.datetime.utcnow()
+        if len(emoji) == 1:
+            for key, value in emoji.items():
+                print(key)
+                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
+                embed.set_thumbnail(url=random.choice(value).url)
+        else:
+            msg = ''
+            for key, value in emoji.items():
+                emoji_choice = random.choice(value)
+                error_msg = f'There are **{len(value)}** emojis in the **{key}** category\n'
+                msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {error_msg}'
+            embed.description = msg
+        log.trace('Emoji embed sent')
+        return embed
+
+    @staticmethod
+    def generate_invalid_embed(ctx: commands.Context) -> discord.Embed:
+        """Genrates error embed."""
+        embed = discord.Embed()
+        embed.color = Colours.soft_red
+        embed.title = "Invalid Input"
+        emoji_dict = {}
+        for emoji in ctx.guild.emojis:
+            emoji_dict.update({emoji.name.split("_")[0]: []})
```

Comment:
This is a lot more clearer, please replace all the similar calls by this. 
```suggestion
            emoji_dict[emoji.name.split("_")[0]] = []
```